CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Bug fixes
 
+- [#808](https://github.com/alleslabs/celatone-frontend/pull/808) Migrate to temp osmosis graphql
```

Comment:
809

Reply:
using 808 the same as PR merging into dev